### DISCUSSION TITLE
Planned release of 0.9

### DISCUSSION THREAD

**Discussion by jorgecarleitao** [jorgecarleitao/arrow2#754] [Ideas]
Hi,

I am planning a release of 0.9 from main this week, and was thinking on supporting it for 6 months. Any opinions about tradeoffs and whether you think it is worth?

Specifically, 0.9 would receive every backward-compatible feature and bug fix for the next 6 months under 0.9.Z, with releases done from a 0.9.Z branch and on every PR that lands on main that would be sufficiently compatible with 0.9.Z (there are limitations to my capacity xD)

**Comment by houqp** [comment]
I think it's not worth your time in the current stage of the project unless someone is paying you for this or doing a decent chunk of the backporting work for you :P

**Comment by ritchie46** [comment]
6 months is a long time. Given that there were quite some backwards incompatible changes lately, I am inclined to go for rapid improvement.

The older the project, the more painful it becomes to break changes. Hopefully we  hit an asymptote naturally.

**Reply by jorgecarleitao** [reply]
My thinking was to continue to release breaking changes on 0.10, 0.11, etc at a fast pace irrespectively of 0.9. I was more thinking about [this issue](https://github.com/grafana/grafana-plugin-sdk-rust/pull/35), cc @toddtreece @sd2k

**Reply by ritchie46** [reply]
I looke ad the Cargo.toml and he was using a 0.7 crates.io released arrow2. 

Doesn't that issue remains if consumers are out of sync?

**Reply by sd2k** [reply]
Hi folks 👋 our SDK is still very much in beta so I'm happy to periodically release new versions with bumped `arrow2` dependencies - that's the (inexpensive) price of being on the cutting edge, I suppose 😄 

Thanks for thinking about us though, @jorgecarleitao! I'll keep an eye on this discussion and others.

**Reply by jorgecarleitao** [reply]
Thanks a lot for your understanding. Let's keep the existing setup for now, then.
